hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -83,11 +83,12 @@ public String getFileName() {
     public HoodieLogFile rollOver(FileSystem fs) throws IOException {
         String fileId = getFileId();
         String baseCommitTime = getBaseCommitTime();
+        String extension = "." + FSUtils.getFileExtensionFromLog(path);
```

Comment:
I did add HoodieArchivedLogFile.java, if you scroll up you'll see it. Some methods don't make sense yes, I'll take another pass at it.